codebuild/release/upload_artifacts.yml

```
@@ -8,13 +8,17 @@ env:
     BRANCH: "master"
   git-credential-helper: yes
   secrets-manager:
-    GH_TOKEN: Github/aws-crypto-tools-ci-bot:personal\ access\ token\ (new\ token\ format)
+    GH_TOKEN: Github/aws-crypto-tools-ci-bot:ESDK Release Token
 
 phases:
   pre_build:
     commands:
         # get new project version
       - export VERSION=$(grep version pom.xml | head -n 1 | sed -n 's/[ \t]*<version>\(.*\)<\/version>/\1/p')
+      - git config --global user.name "aws-crypto-tools-ci-bot"
+      - git config --global user.email "no-reply@noemail.local"
+      - echo $GH_TOKEN > token.txt
+      - export GH_TOKEN=
```

```suggestion
      - export -n GH_TOKEN
```
The above assumes that terminal CodeBuild provides us is bash. If it is `zsh`, my suggestion will fail.
But it is not: https://docs.aws.amazon.com/codebuild/latest/userguide/build-spec-ref.html#build-spec.shell
And both `sh` and `bash` support the `-n` flag on `export`.

---

ooh i like that better it is much cleaner. i'll test this and get back to you if we still have the same result

---

big oof! unfortunately the `export -n` option returns with `export: Illegal option -n`

---

For the record, CodeBuild does not support `-n` on `export`. We could probably use `unset`, but testing this is manual and time consuming, so I am approving this as is.